library/src/main/java/com/chuckerteam/chucker/internal/data/entity/HttpCookie.kt

```diff
@@ -0,0 +1,26 @@
+package com.chuckerteam.chucker.internal.data.entity
+
+internal data class HttpCookie(val name: String, val value: String) {
+    companion object {
+        fun fromRequestHeader(cookieHeader: String?): List<HttpCookie> =
+            mutableListOf<HttpCookie>().apply {
+                if (!cookieHeader.isNullOrEmpty()) {
+                    cookieHeader.split(";").forEach { c ->
+                        val (name, value) = c.split("=")
```

Comment:
I believe these functions shouldn't be here. 
Also, the code could be simplified, since current one is cool from usage of Kotlin features point of view, but hard to read.